### DISCUSSION TITLE
Italian spell checker

### DISCUSSION THREAD

**Discussion by GitGianluc** [URUWorks/TeroSubtitler#94] [Q&A] [answered]
Is it possible to add the [Italian dictionary](https://github.com/URUWorks/TeroSubtitler/files/11665152/italian-dictionary.zip) for spell checking?

Credits:  [LibreOffice Italian Spellchecker built by PLIO](https://extensions.libreoffice.org/en/extensions/show/italian-dictionary-thesaurus-hyphenation-patterns)

**Comment by URUWorks** [comment, marked as answer]
Yes, you can still incorporate it by copying the dictionary files (*.aff/*.dic) into the "dictionaries" folder of "tero subtitler.app".

**Reply by GitGianluc** [reply]
Wow, thanks for pointing this out!

**Comment by URUWorks** [comment]
Added v1.0.0.2
